src/libs/safe-path/src/scoped_dir_builder.rs

```diff
@@ -232,7 +233,6 @@ mod tests {
 
     #[test]
     fn test_create_root() {
-        // root directory doesn't exist
```

Comment:
This change should be folded into the first patch.

Reply:
Fixed